packages/components/src/components/icon-trade-types/icon-trade-types.tsx

```diff
@@ -1,91 +1,89 @@
 import React from 'react';
 import Icon from '../icon/icon';
 import { TIconProps } from '../types';
-import { VANILLALONG, TURBOS } from '@deriv/shared';
+import { CONTRACT_TYPES } from '@deriv/shared';
 
 type TIconTradeTypes = Omit<TIconProps, 'icon'> & {
     type: string;
     className?: string;
 };
 
 const IconTradeTypes = ({ type, className, ...props }: TIconTradeTypes) => {
-    switch (type.toLowerCase()) {
```

Comment:
It turns out that `type` here was passed as string in lowercase. No need in addition adding `.toLowerCase()` + I replaced with `.toUppeCase()` in order to use created constants

Reply:
No, it's written in lower case by default

Reply:
I double checked and print `type` in the console: 
<img width="730" alt="Screenshot 2023-11-13 at 09 18 53" src="https://github.com/binary-com/deriv-app/assets/121025168/0028d927-41ce-4711-b1f1-66320b0df0af">
<img width="730" alt="Screenshot 2023-11-13 at 09 13 14" src="https://github.com/binary-com/deriv-app/assets/121025168/94819f02-0754-45fc-a28c-52efdef36d4c">
It is written in lower case by default for all contracts